packages/core/src/Stores/traders-hub-store.js

```diff
@@ -35,6 +35,7 @@ export default class TradersHubStore extends BaseStore {
     active_modal_wallet_id;
     is_cfd_restricted_country = false;
     is_financial_restricted_country = false;
+    is_setup_real_account_or_go_to_demo_modal_visible = false;
```

Comment:
@sergei-deriv add this flag to `should_show_passkeys_info_modal` in `app-modals` in `core` package